manim/utils/config_utils.py

```diff
@@ -18,6 +18,7 @@
 
 __all__ = ["_run_config", "_paths_config_file", "_from_command_line"]
```

Comment:
NitPick: You missed `curr_config_dict` here.

Reply:
Good catch! I'll add it in.